### ISSUE TITLE
ACR build fails with generic nodeType object is not callable

### ISSUE THREAD

**Issue by paulsalmon-za** [Azure/azure-cli#18773]
Running the command on my local machine with the latest Azure CLI as well as a GitHub action https://github.com/marketplace/actions/azure-container-registry-build on a free account

## Describe the bug

**Command Name**
`az acr build`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
'NoneType' object is not callable
Traceback (most recent call last):
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\knack/cli.py", line 231, in invoke
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 657, in execute
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 720, in _run_jobs_serially
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 691, in _run_job
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 328, in __call__
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/command_operation.py", line 121, in handler
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/command_modules/acr/build.py", line 106, in acr_build
TypeError: 'NoneType' object is not callable

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az acr build -r {} -f {} -t {} {}`

## Environment Summary
```
Windows-10-10.0.19041-SP0
Python 3.8.9
Installer: MSI

azure-cli 2.26.0

```

**Comment by yonzhan** [comment]
route to service team

**Comment by Tsongtsz** [comment]
I have the same issue here. Any solutions?

**Comment by paulsalmon-za** [comment]
> I have the same issue here. Any solutions?

I required it to setup a CICD pipeline in GitHub, I changed the pipeline to use the docker/build-push-action action and using the azure/docker-login action to authenticate the docker registry

**Comment by northtyphoon** [comment]
The issue has been fixed since 2.26.1
